### PR TITLE
Add Treatment Goals to Scenario Config + Corresponding Tests Update

### PR DESCRIPTION
**Pull request by sachiyuan**
- Added treatment goal questions to scenario config page 
- Updated Corresponding tests 
  - Tests that are now unnecessary have been deleted 
    - e.g tests concerned with behavior of priority table (replaced with treatment goal questions)
  - Tests that will need to be updated once the rest of the scenario configs have been added have been temporarily commented out

Screenshot 2023-08-10 at 4 09 41 PM

### CODE REVIEW

File: src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts [hunk 1]
```diff
@@ -64,7 +64,11 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
   panelExpanded: boolean = true;
   stepStates: StepState[];
   treatmentGoals: Observable<TreatmentGoalConfig[] | null>;
-
+  defaultQ: TreatmentQuestionConfig = {
```
Comment (runforyourlives) [inline src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts:67]: I know it sounds silly, but write out Question.

**Review by runforyourlives — APPROVED** [review]
Approved conditionally - let's chat with the group tomorrow morning to make sure that we're ok with freezing demo.planscape.org for a while until the scenario page is fully back.

**Review by runforyourlives — CHANGES_REQUESTED** [review]
actually, "conditional approval" means nothing.  :P Let's bring this up with the group in the morn.

**Review by runforyourlives — APPROVED** [review]
GTG

File: src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts (hunk 1 above)
Comment (sachiyuan) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts]: Whoops thanks for catching that!

**Pull request merged**